Modules/Core/ImageFunction/include/itkGaussianDerivativeImageFunction.h

```diff
@@ -75,23 +75,23 @@ class ITK_TEMPLATE_EXPORT GaussianDerivativeImageFunction:
   using IndexType = typename InputImageType::IndexType;
 
   /** Dimension of the underlying image. */
-  static constexpr unsigned int ImageDimension2 = InputImageType::ImageDimension;
+  static constexpr unsigned int ImageDimension = InputImageType::ImageDimension;
```

Comment:
ImageDimension2 should be available for backwards compatibility only visible when LEGACY is turned on.